e2e-tests/playwright/e2e/learning-path-page.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { expect, test } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { Common } from '../utils/Common';
+
+test.describe('Learning Paths', () => {
+  let common: Common;
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    uiHelper = new UIhelper(page);
+    common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test('Verify that links in Learning Paths for Backstage opens in a new tab', async ({
+    page,
+  }) => {
+    await uiHelper.openSidebar('Learning Paths');
+
+    for (let i = 0; i < 5; i++) {
+      const popupPromise = page.waitForEvent('popup');
+      await page.locator(`div[class*="MuiCardHeader-root"]`).nth(i).click();
+      const popup = await popupPromise;
+      await popup.waitForLoadState('domcontentloaded');
```

Comment:
@teknaS47 its failing consistently here, can you try handling pop like : https://github.com/janus-idp/backstage-showcase/blob/main/e2e-tests/playwright/utils/Common.ts#L103-L120